app/js/services/filter.service.js

```diff
@@ -56,7 +56,8 @@ function FilterService(
 
     filterService.setAge = function (age) {
         _age = Number.parseInt(age);
-        filterService.setQueryParam('age', age);
+        _age = (_age === 0) ? null : _age;
+        filterService.setQueryParam('age', _age);
```

Comment:
this is setting the age param to null when you select 'All' from the age select which is causing the server to throw a 404. This can be fixed by checking for null here:

https://github.com/RedHatInsights/insights-frontend/blob/features/newestRulesOverview/app/js/services/filter.service.js#L634

![screen shot 2017-11-06 at 1 35 22 pm](https://user-images.githubusercontent.com/5181010/32457509-6178a49c-c2f7-11e7-9f8c-45a489c30704.png)